modules/internal/ChapelIO.chpl

```diff
@@ -880,7 +872,7 @@ module ChapelIO {
      if any, then exits the program.
    */
   proc halt() {
-    __primitive("chpl_error", "halt reached");
+    __primitive("chpl_error", c"halt reached");
```

Comment:
I think we should enumerate user-facing changes to the team before merging so that nobody is caught by surprise.  The C-string literal syntax is an example.  What else?  (this question is what motivated me to browse this PR)